frontend/src/pages/genre/_itemId/index.vue

```diff
@@ -57,29 +60,54 @@ const items = itemsStore();
 const route = useRoute();
 const remote = useRemote();
 
-const itemId = route.params.itemId;
-const item = (
-  await remote.sdk.newUserApi(getUserLibraryApi).getItem({
-    userId: remote.auth.currentUserId || '',
-    itemId
-  })
-).data;
-
-let genres = (
-  await remote.sdk.newUserApi(getItemsApi).getItems({
-    genreIds: [item.Id as string],
-    includeItemTypes: [route.query.type.toString()],
-    recursive: true,
-    sortBy: ['SortName'],
-    sortOrder: [SortOrder.Ascending],
-    fields: Object.values(ItemFields),
-    userId: remote.auth.currentUserId || ''
-  })
-).data.Items;
-
-genres = items.addCollection(item, genres || []);
-
-route.meta.title = item.Name;
+const loading = ref(false);
+const genre = ref<BaseItemDto>({});
+const genres = ref<BaseItemDto[]>([]);
+
+onMounted(async () => {
```

Comment:
Same as above, I don't think we need the loading state in these components at all now. These components had this way of loading because they're SSR remnants